test/evergreen.yml

```diff
@@ -4134,8 +4190,14 @@ tasks:
             set -o errexit
             set -o verbose
             ${PREPARE_TEST_ENV}
+            # Check if we running in disagg mode.
```

Comment:
should call this in disagg buuild variant.

Reply:
Nice catch!